File: app/src/main/java/com/example/petrsumobile/schedule/Schedule.java
package com.example.petrsumobile.schedule;

public class Schedule {

    private String classroom;
    private String startTime;
    private String endTime;
    private String lecturer;
    private String title;
    private String type;
    private Integer number;

    public Schedule(String classroom,String startTime,
                    String endTime,String lecturer,
                    String title, String type, Integer number){
        this.classroom = classroom;
        this.startTime = startTime;
        this.endTime = endTime;
        this.lecturer = lecturer;
        this.title = title;
        this.type = type;
        this.number = number;
    }

    public String getClassroom() {
        return classroom;
    }

    public String getStartTime() {
        return startTime;
    }

    public String getEndTime() {
        return endTime;
    }

    public String getLecturer() {
        return lecturer;
    }

    public String getTitle() {
        return title;
    }

    public String getType() {
        return type;
    }

    public Integer getNumber() {
        return number;
    }
}

File: app/src/main/java/com/example/petrsumobile/schedule/Schedule.kt
package com.example.petrsumobile.schedule

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



data class ScheduleWeekType(
        @Expose
        val denominator : List<List<Schedule>>,
        @Expose
        val numerator : List<List<Schedule>>)

data class ScheduleWeek(var scheduleList: List<Schedule>,
                        var nameWeek: String)

@Entity
data class Schedule(
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,
        @Expose
        var classroom: String,
        @Expose
        @SerializedName("start_time")
        var startTime: String,
        @Expose
        @SerializedName("end_time")
        var endTime: String,
        @Expose
        var lecturer: String,
        @Expose
        var title: String,
        @Expose
        var type: String,
        @Expose
        var number: Int,
        @Expose
        var group: Int)